build.sbt

```diff
@@ -2,7 +2,7 @@ name := "strategygames"
 
 organization := "org.playstrategy"
 
-version := "10.2.1-pstrat135"
+version := "10.2.1-pstrat136-dev00"
```

Comment:
You'll want to make sure that when you merge this to dev that you rebuild SG from dev and make all the places (lila, lila-ws, mindtrap etc) that use it take the new version. There is stuff on dev that wont be in your `pstrat136-dev00` version.
Currently the way I'm working is to not commit a change to the version number on my local branch, but get the merge to dev done and then up the version on the dev branch and commit that directly and build from dev. that way you know you aren't losing anyones work/changes